src/query/v2/conversions.hpp

```diff
@@ -56,6 +56,10 @@ inline TypedValue ValueToTypedValue(const msgs::Value &value, msgs::ShardRequest
   throw std::runtime_error("Incorrect type in conversion");
 }
 
+inline const auto ValueToTypedValueFunctor = [](const msgs::Value &value, msgs::ShardRequestManagerInterface *manager) {
+  return ValueToTypedValue(value, manager);
+};
```

Comment:
Isn't `detail::Callable` does the same in `src/query/v2/bindings/eval.hpp`? If so, then please don't duplicate the same functionality.

Reply:
Yes but unfortunately that file depends on the generated ast which will blow up if you include it. I will figure out some other way but not in this PR. If you have a better idea let me know!

Reply:
`detail::Callable` doesn't depend on any generated thing, so it can be extracted to a file from which you can include in both places.

Reply:
`detail::Callable` does more thing that this function, so it is okay to not to unify them.